### PR TITLE
02-user-can-view-the-average-grade- Michael C

### PR DESCRIPTION
**Pull request by michael5522**
Screen Shot 2020-05-09 at 11 42 05 AM

### CODE REVIEW

File: components/app.js [hunk 1]
```diff
@@ -10,6 +11,12 @@ class App {
   }
   handleGetGradesSuccess(grades){
     this.gradeTable.updateGrades(grades);
+    var tempGrade = 0;
+    for(var i =0; i<grades.length;i++){
+      tempGrade += grades[i][`grade`];
```
Comment (WhiskeyJack96) [inline components/app.js]: grades[i].grade should work any reason you indexed the object with a string?

**Review by WhiskeyJack96 — CHANGES_REQUESTED** [review]
One small change

File: components/app.js (hunk 1 above)
Comment (michael5522) [reply components/app.js]: I tried tempGrade += grades[i][grades] at first but it didnt work.
so I tried this one and it worked tempGrade += grades[i][`grade`]

**Review by WhiskeyJack96 — APPROVED** [review]
Great!

**Pull request merged**